test/e2e/tests/request-queuing/sendTx-revokePermissions.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import TestDapp from '../../page-objects/pages/test-dapp';
+import TransactionConfirmation from '../../page-objects/pages/confirmations/redesign/transaction-confirmation';
+import { Ganache } from '../../seeder/ganache';
+import { Driver } from '../../webdriver/driver';
+
+import FixtureBuilder from '../../fixture-builder';
+import {
+  withFixtures,
+  defaultGanacheOptions,
+  WINDOW_TITLES,
+} from '../../helpers';
+
+describe('Request Queuing', function () {
+  // TODO: add a new spec which checks that after revoking and connecting again
+  // a pending tx is still closed when using revokePermissions.
+  // To be done once this bug is fixed: #29272
```

Comment:
a new bug has been discovered while migrating this spec.
Once the bug is fixed, we should add a new spec to verify that flow too.
Basically the first time we perform the flow it works as expected, but if we re-conenct to the dapp, trigger a new tx, and revoke permissions, that new tx is not cleared.